docs/Available-Symbols-Generators.md

```diff
@@ -106,12 +105,16 @@ In this sample three symbols are defined:
 ## Coalesce
 Behaves like the C# `??` operator.
 
+Note: as in C# `??` operator, empty string will be accepted as the valid value and returned. 
+The typical use of this generator is to check if the parameter was provided by user, otherwise set fallback generated value.
+
 #### Parameters
 
-| Name     |Data Type| Description   |
-|----------|------|---------------|
-|sourceVariableName|string|source of the data|
-|fallbackVariableName|string|alternate source of data|
+|Name|Data Type|Description|Mandatory|
+|---|---|---|---|
+|`sourceVariableName`|`string`|the symbol name which is a primary source of data (left operand of `coalesce`)|yes|
+|`fallbackVariableName`|`string`|the symbol name which is an alternate source of data(right operand of `coalesce`)|yes|
+|`defaultValue`|`string`|The default value. In case it is specified, and primary source is equal to this value, the fallback value will be used.|no|
```

Comment:
It as well serves as a 'fallback fallback' (https://github.com/dotnet/templating/pull/5223/files#diff-aeca7dd6cf4839fd716aadea060367dc50538b7f004eb2da15eb1bf7abcce207R35) - should it be explicitly mentioned in the doc? Or is it already too confusing and we'll leave it as a hidden feature?